src/bindings/js/node/tests/unit/infer_request.test.js

```diff
@@ -300,3 +300,47 @@ describe('ov.InferRequest tests', () => {
     });
   });
 });
+
+describe('ov.InferRequest tests with missing outputs names', () => {
+  const modelV3Small = testModels.modelV3Small;
+  let compiledModel = null;
+  let tensorData = null;
+  let tensor = null;
+  let inferRequest = null;
+
+  before(async () => {
+    await isModelAvailable(modelV3Small);
+
+    const fs = require('fs');
+    const core = new ov.Core();
+
+    let model_data = fs.readFileSync(getModelPath(modelV3Small).xml, 'utf8');
+    const weights = fs.readFileSync(getModelPath(modelV3Small).bin);
+    model_data = model_data.replace("names=\"MobilenetV3/Predictions/Softmax:0\"", "");
+    const model = core.readModelSync(Buffer.from(model_data, 'utf8'), weights);
+
+    compiledModel = core.compileModelSync(model, 'CPU');
+    inferRequest = compiledModel.createInferRequest();
+
+    tensorData = Float32Array.from(
+      { length: 150528 },
```

Comment:
"Magic number". Propose to calculate it on the flight, base on the model input shape.

Reply:
Yes, just like it is used in other tests. This is fixed in another PR with refactoring.